ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.17__multi-nightly.yaml

```
@@ -1319,6 +1319,20 @@ tests:
     test:
     - chain: openshift-e2e-test-qe
     workflow: cucushift-installer-rehearse-gcp-ipi-capability-additionalcaps
+- as: gcp-ipi-c3-metal-amd-f999
```

```suggestion
- as: gcp-ipi-c3-metal-amd-f60
```

`f999` means this job is disabled, will be removed in a short future.

---

By "f60", do you really mean to run the test every 2 months? 

cc @yunjiang29 @gpei

---

`f60` is the max frequency we supported I know.

---

@jianli-wei do we have the plan to run the c3-metal testing in regular CI?

---

@jianlinliu When we talked about adding such tests in PROW CI, with @yunjiang29 and @gpei, we'd thought to trigger the tests manually (e.g. via API), rather than let the tests run periodically, on considering the cost. 

@liangxia Is it possible to not remove the tests, e.g. `gcp-ipi-c3-metal-amd-f999` / `gcp-ipi-c3-metal-amd-f999-destructive`?

---

We don't remove f999 jobs unless the given version reach EOL.

---

If we agreed with that we trigger the regular ci by remote API , I have no objection. But if we only care about one-shot testing, especially installation, I am thinking if it is deserved to be a regular ci job, why not a installer team dedicated ci job.

---

The tests aim to include both installation test and E2E test, so we add them as regular ci jobs.